monkeytype/typing.py

```diff
@@ -251,8 +251,18 @@ def rewrite(self, typ):
         return typ
 
 
+class RewriteGenerator(TypeRewriter):
+    """Returns an Iterator, if the send_type and return_type of a Generator is None"""
+
+    def visit_Generator(self, typ):
+        args = typ.__args__
+        if args[1] is None and args[2] is None:
+            return Iterator[args[0]]
+
```

Comment:
This should return `typ` in the case where no rewrite occurs.

Reply:
I have fixed the broken tests. Thanks